### PR TITLE
Traefik metrics

### PR DESCRIPTION
**Pull request by jac**
Enabled Traefik Metrics; Port: 8082; Endpoint: /metrics

Added swarm managers to `prometheus_traefik` group

Reverse proxied metrics using exporter exporter

Added `prometheus_traefik` group to Prometheus scrape configs

### CODE REVIEW

File: roles/traefik-proxmox/tasks/main.yml [hunk 1]
```diff
@@ -72,6 +72,10 @@
                 published: 443
                 protocol: tcp
                 mode: host
+              - target: 8082
+                published: 8082
+                protocol: tcp
+                mode: host
```
Comment (Strum355) [inline roles/traefik-proxmox/tasks/main.yml:78]: This will publicly expose the metrics endpoint. Is it protected otherwise somehow that ive missed?

Comment (jac) [reply roles/traefik-proxmox/tasks/main.yml:78]: I'd need to double check with @ocanty but from WAN only ports 80 and 443 get forwarded to Traefik



https://github.com/UCCNetsoc/NaC/blob/master/provision-router.yml#L68

Comment (ocanty) [reply roles/traefik-proxmox/tasks/main.yml:78]: this is correct

**Comment by ocanty** [comment]
lgtm

**Pull request merged**